### PR TITLE
Wakawaka Mapmaaaan~

### PR DESCRIPTION
**Pull request by Alkallid**
What's a crate, on Helheim?
'-ter' instead of '-tre' I think is used in vanilla- you might want to change that.
And programme.
Will you be filling in the spaceports more?

### CODE REVIEW

File: data/map.txt [hunk 1]
```diff
@@ -3930,12 +3930,13 @@ planet Forefather
...
+	description `When the first settlers landed on Frodo, they found the habitable zones of the planet to be remarkably similar to the "Shire" from 'The Lord of the Rings', an infamous book from the 20th century. As a result, they decided to name the planet Frodo, after the book's main character.`
+	description `	While most of the population here consists of farmers, taking care of and managing the countryside and living in rustic homes, a surprisingly large percentage of inhabitants find this world as dull as an old, musty sock, and are scraping together all the money they can possibly find to leave this planet far behind.`
 	spaceport `The spaceport is extremely flashy for a planet like this. Although it does not use sweeping, graceful curves like many other spaceports do, it still has an atmosphere of class and optimism about it, all through the white-stained granite blocks that make up most of the buildings.`
 	spaceport `	The air here smells very similar to your home planet of New Boston.`
 	shipyard "XSZ Tier 1"
 	outfitter "XSZ Basics"
+#Why INfamous?
```
Comment (Storm-Crow01) [inline data/map.txt:3939]: Who knows what could happen in an entire millenium?

**Comment by Alkallid** [comment]
Why would you name a giant home and industry for a midget whom everyone hates?

**Comment by Storm-Crow01** [comment]
Since most of this galaxy was settled by 2550, there's still potential that LOTR could have become hated in vanilla human space. 400 years is a while.

**Comment by Alkallid** [comment]
Inb4 orbital bombardment by Enclave for nerdy names

**Comment by Storm-Crow01** [comment]
.>the enclave are basically deep 2.0
.>the deep are just one big group of sciencey nerds
.>get rekt m90

**Comment by Alkallid** [comment]
You're a giant space nerd.

**Comment by Storm-Crow01** [comment]
Why thank you c:

**Comment by Alkallid** [comment]
Ew

**Comment by Storm-Crow01** [comment]
Heheheh. Anyway, thanks for the spell checking; I'll go ahead and merge this.

**Comment by Alkallid** [comment]
Ayy

**Pull request merged**